src/main/scala/subsystem/Configs.scala

```diff
@@ -271,6 +271,12 @@ class WithNBreakpoints(hwbp: Int) extends Config ((site, here, up) => {
   }
 })
 
+class WithHypervisor(hext: Boolean = true) extends Config ((site, here, up) => {
```

Comment:
Sorry for nit, please remove extra space
```suggestion
class WithHypervisor(hext: Boolean = true) extends Config((site, here, up) => {
```